src/test/resources/samples/json_merge/full/existing.jsonl

```diff
@@ -0,0 +1,78 @@
+{
```

Comment:
Hi @AndreiBordak, 

It seems like you're using the response from the GET /resource/{id} API as the content for this file. Let's not do that. The API response has a custom structure for rendering our UI, which doesn't reflect the structure of a Resource node's document in the graph.
The 'existing' and 'incoming' documents will always follow this structure:

```
{
    "key1": [
         "value1",  // values will be always strings.
         "value2"
    ], 
    "key2": [
         "value1", 
         "value2"
    ]
}
```

We could create a different test case to ensure that the application will not break just in case if a different structure of document is received as input. But our main focus should be to test the document structure I have given above.

Reply:
Sure PK

Reply:
Done.